src/Client.php

```diff
@@ -455,6 +456,46 @@ public function getBulkLabelPdf(array $parcels, int $format): string
         }
     }
 
+
+    /**
+     * Fetches a single parcel document based on requested document type, content type, and dpi.
+     *
+     * If the parcel hasn't had its labels created through {@see self::createLabel()} it will result in an exception.
+     *
+     * @param string $documentType One of `Parcel::DOCUMENT_TYPES`
+     * @param string $contentType One of `Parcel::CONTENT_TYPES`
+     * @param int $dpi One of `Parcel::DPI_VALUES` limited by the selected `$contentType`
```

Comment:
Tip: You can use `@see` to have IDEs (at least the Jetbrains ones) link to the types and allow easier refactors!

```suggestion
     * @param string $documentType One of {@see Parcel::DOCUMENT_TYPES}
     * @param string $contentType One of {@see Parcel::CONTENT_TYPES}
     * @param int $dpi One of {@see Parcel::DPI_VALUES} limited by the selected `$contentType`
```